src/libraries/Microsoft.Extensions.Logging.Console/src/SimpleConsoleFormatter.cs

```diff
@@ -0,0 +1,211 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I think Toub has been updating all the copyright headers to a new 2 line copyright. You should check all the other files in the repo for the new format.